packages/calypso-build/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@automattic/calypso-build",
+	"version": "1.0.0",
+	"description": "Shared build configurations.",
+	"keywords": [
+		"wordpress"
+	],
+	"author": "Automattic",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"src"
```

Comment:
We renamed this dir to `webpack/`. Best to drop the entire `files:` stanza (so we automatically include all files in the package, including e.g. `.eslintrc.js`).

Reply:
e7471bab05